lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -19,8 +20,11 @@
 from course_modes.models import CourseMode
 from lms.djangoapps.course_api.blocks.transformers.blocks_api import BlocksAPITransformer
 from lms.djangoapps.course_blocks.api import get_course_block_access_transformers, get_course_blocks
+from lms.djangoapps.course_goals.api import add_course_goal, get_course_goal, get_course_goal_text, \
+    has_course_goal_permission, valid_course_goals_ordered
```

Comment:
It's more common in our code to use parantheses rather than the backslash for this.